### ISSUE TITLE
How to recognize user by Authorize token

### ISSUE THREAD

**Issue by cenxui** [aws/aws-lambda-go#106]
Thank you for reading this issue, the scenario is below.

User send a request by jwt.Token belong to Cogntio User Pool to API Gateway which is authorized but cannot recognize the request belonge to which user.
The "events.APIGatewayProxyResponse"  doesn't provide information by default like there openId, email, etc. 

How could I recognize user by Authorize token?

**Comment by piotrkubisa** [comment]
As you know, in the payload of the AWS Lambda event there isn't much data about who exactly was trying to get some resource from your API while identifing via `Authorization` header. Thanks to that Lambda "acts" universally, I mean you are not obliged to use AWS Cognito service or API key. If you are using Cognito with a JWT token, in my opinion **you should parse it in your lambda handler, verify it and check claims** for more data. Sadly, right now I cannot provide you code snippet for easy copy&paste but I think it will be quite easy for you to develop own algorithm using https://github.com/dgrijalva/jwt-go package when you follow instruction on AWS Knowledge Center https://aws.amazon.com/premiumsupport/knowledge-center/decode-verify-cognito-json-token/

tl;dr: bolded
